recipe/meta.yaml

```diff
@@ -90,7 +90,7 @@ outputs:
         - python >=3.9
         - nglview
         - notebook
-        - openff-nagl >=0.3.1
+        - openff-nagl >=0.3.7
```

Comment:
`=0.4` might be safer depending on what other DGL stuff is brought down and if lookup tables are the expected behavior in the example(s)

https://github.com/openforcefield/openff-nagl/blob/main/docs/CHANGELOG.md#v040----2024-07-18

https://github.com/openforcefield/openff-nagl/pull/117/files#r1622912084

Reply:
I'm not sure about the root cause here, but during releases I generally look for [recent changes in our testing envs](https://github.com/openforcefield/openff-toolkit/blame/main/devtools/conda-envs/test_env.yaml#L23) and mirror those into the recipe.

Reply:
And the lookup tables just had their first release very recently, and I'm not sure whether any of the Toolkit tests/examples touch them. In case there's something funny with the 0.4 series, I'd like to give users the opportunity to downgrade.